src/containers/EventForm/components/ArtworkForm/function.js

```diff
@@ -0,0 +1,28 @@
+import { ArtworksRepository, PeopleRepository } from 'hackoss';
+import { FirebaseApp } from 'utils/firebase';
+
+async function getPeopleBase() {
+  const pplRepo = new PeopleRepository(FirebaseApp);
+  const people = await pplRepo.getPeople();
+  return people;
+}
+export async function getPeople() {
+  const people = await getPeopleBase();
+  return people.map(person => person.name);
+}
+
+export const createArtwork = setStatus => async input => {
+  const pplRepo = new PeopleRepository(FirebaseApp);
```

Comment:
Okay, now I understand where you were trying to get at ntuoss/hackoss#11. Would it help to write the instances for those repositories in a separate file and then just import from there? Check out this snippet from ntuoss/hackoss-admin:

```ts
import { EventsRepository, FirebaseRepository, PeopleRepository, LocationsRepository, OrganisationsRepository, ArtworksRepository } from 'hackoss';
import { environment } from '../environments/environment';

// singleton instances
export const fr = new FirebaseRepository(environment.firebase);
export const pr = new PeopleRepository(fr);
export const lr = new LocationsRepository(fr);
export const or = new OrganisationsRepository(fr);
export const ar = new ArtworksRepository(fr, pr);
export const er = new EventsRepository(fr, pr, lr, or, ar);
```

Then in the components where u need it, you just import and use the instance directly.

Reply:
Good fix! Will implement later.